src/index/vespa_.py

```diff
@@ -105,10 +111,31 @@ class VespaFamilyDocument(BaseModel):
     document_content_type: Optional[str] = None
     document_cdn_object: Optional[str] = None
     document_source_url: Optional[str] = None
+    document_title: Optional[str] = None
+    family_geographies: Optional[list[str]] = None
+    corpus_import_id: Optional[str] = None
+    corpus_type_name: Optional[str] = None
+    collection_title: Optional[str] = None
+    collection_summary: Optional[str] = None
+    metadata: list[MetadataItem]
```

Comment:
**fix:** I just realised that we should've made this optional, similar to the other fields?
```suggestion
    metadata: Optional[list[MetadataItem]]
```

Reply:
Ah yes, thank you